classes/orcid/OrcidReview.php

```diff
@@ -0,0 +1,128 @@
+                'reviewer-role' => 'reviewer',
+                'review-type' => 'review',
+                'review-completion-date' => [
+                    'year' => [
+                        'value' => $reviewCompletionDate->format('Y')
+                    ],
+                    'month' => [
+                        'value' => $reviewCompletionDate->format('m')
+                    ],
+                    'day' => [
+                        'value' => $reviewCompletionDate->format('d')
+                    ]
+                ],
+                'review-group-id' => 'issn:' . $this->context->getData('onlineIssn'),
+
+                'convening-organization' => [
+                    'name' => $this->context->getData('publisherInstitution'),
+                    'address' => [
+                        'city' => OrcidManager::getCity($this->context),
+                        'country' => OrcidManager::getCountry($this->context),
+
+                    ]
+                ],
+                'review-identifiers' => ['external-id' => [
+                    [
+                        'external-id-type' => 'source-work-id',
+                        'external-id-value' => $this->review->getData('reviewRoundId'),
+                        'external-id-relationship' => 'part-of']
+                ]]
+            ];
+            if ($this->review->getReviewMethod() == ReviewAssignment::SUBMISSION_REVIEW_METHOD_OPEN) {
+                $orcidReview['subject-url'] = ['value' => $publicationUrl];
+                $orcidReview['review-url'] = ['value' => $publicationUrl];
+                $orcidReview['subject-type'] = 'journal-article';
+                $orcidReview['subject-name'] = [
+                    'title' => ['value' => $this->submission->getCurrentPublication()->getLocalizedData('title') ?? '']
+                ];
+
+                if (!empty($currentPublication->getDoi())) {
+                    /** @var Doi $doiObject */
+                    $doiObject = $currentPublication->getData('doiObject');
+                    $externalIds = [
+                        'external-id-type' => 'doi',
+                        'external-id-value' => $currentPublication->getDoi(),
+                        'external-id-url' => [
+                            'value' => $doiObject->getResolvingUrl(),
+                        ],
+                        'external-id-relationship' => 'self'
+
+                    ];
+                    $orcidReview['subject-external-identifier'] = $externalIds;
+                }
+            }
+
+            $allTitles = $currentPublication->getData('title');
+            foreach ($allTitles as $locale => $title) {
+                if ($locale !== $submissionLocale) {
+                    $orcidReview['subject-name']['translated-title'] = ['value' => $title, 'language-code' => LocaleConversion::getIso1FromLocale($locale)];
+                }
+            }
+
+            return $orcidReview;
+        } else {
```

Comment:
We could also put this condition at the beginning, so that we have one indentation less above